docs/mvvm/MigratingFromMvvmLight.md

```diff
@@ -0,0 +1,842 @@
+
+#### Set<T> ( ref T, T, bool, string )
+
+`Set<T>(ref T, T, bool, string)` has a renamed direct replacement, `SetProperty<T>(ref T, T, bool, string)`. 
+
+```csharp
+// MvvmLight
+this.Set(ref this.myProperty, value, true, nameof(this.MyProperty));
+
+// Toolkit.Mvvm
+this.SetProperty(ref this.myProperty, value, true, nameof(this.MyProperty));
+```
+
+Note, string parameter is not required if the method is being called from the property's setter as it is inferred from the caller member name.
+
+#### Set<T> ( Expression, ref T, T, bool )
+
+`Set<T>(Expression, ref T, T, bool)` does not have a direct replacement. 
+
+It is recommended for improved performance that you replace this with the Toolkit's `SetProperty<T>(ref T, T, bool, string)` using the `nameof` keyword instead.
+
+```csharp
+// MvvmLight
+this.Set<MyObject>(() => this.MyProperty, ref this.myProperty, value, true);
+
+// Toolkit.Mvvm
+this.SetProperty<MyObject>(ref this.myProperty, value, true, nameof(this.MyProperty));
+```
+
+Note, string parameter is not required if the method is being called from the property's setter as it is inferred from the caller member name.
+
+#### Broadcast<T> ( T, T, string )
+
+`Broadcast<T>(T, T, string)` has a direct replacement which doesn't require a rename. 
+
+```csharp
+// MvvmLight
+this.Broadcast<MyObject>(this.oldValue, this.newValue, nameof(this.MyProperty));
+
+// Toolkit.Mvvm
+this.Broadcast<MyObject>(this.oldValue, this.newValue, nameof(this.MyProperty));
+```
+
+Note, the message sent via the `Messenger` property when calling the `Broadcast` method has a direct replacement for `PropertyChangedMessage` within the Toolkit's framework. 
+
+Also, string parameter is not required if the method is being called from the property's setter as it is inferred from the caller member name.
+
+#### RaisePropertyChanged<T> ( string, T, T, bool )
+
+There is no direct replacement for the `RaisePropertyChanged<T>(string, T, T, bool)` method. 
+
+The simplest alternative is to call `OnPropertyChanged` and subsequently call `Broadcast` to achieve this functionality.
+
+```csharp
+// MvvmLight
+this.RaisePropertyChanged<MyObject>(nameof(this.MyProperty), this.oldValue, this.newValue, true);
+
+// Toolkit.Mvvm
+this.OnPropertyChanged<MyObject>(nameof(this.MyProperty));
+this.Broadcast<MyObject>(this.oldValue, this.newValue, nameof(this.MyProperty));
+```
+
+#### RaisePropertyChanged<T> ( Expression, T, T, bool )
+
+There is no direct replacement for the `RaisePropertyChanged<T>(Expression, T, T, bool)` method.
+
+The simplest alternative is to call `OnPropertyChanged` and subsequently call `Broadcast` to achieve this functionality.
+
+```csharp
+// MvvmLight
+this.RaisePropertyChanged<MyObject>(() => this.MyProperty, this.oldValue, this.newValue, true);
+
+// Toolkit.Mvvm
+this.OnPropertyChanged<MyObject>(nameof(this.MyProperty));
```

Comment:
```suggestion
this.OnPropertyChanged(nameof(this.MyProperty));
```

Reply:
Hi Sergio,

Apologies in advance if this is the wrong place to post this – I’m a bit of a GitHub newbie.

I’m trying to migrate an app From MVVMLight. I’ve watched your videos where there is mention of this migration guide, but I cannot find the guide itself, only these change suggestions. Please can you give me a link to the document itself?

The main reason I’m looking for it is to see how to replace the following call I have in the MVVMLight app:

        RaisePropertyChanged (() => EditableEntity);

which triggers an update for an object not a property. How do I do this with this new toolkit?

Thanks in advance,
Peter